<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WUpdateProgress</class>
 <widget class="QDialog" name="WUpdateProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log data period configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#WUpdateProgress {
 background-color:white;
 border-radius: 5px;
}
#label{
 color: #3E4072;
 font-weight: 800;
}
#buttonBox{
 background-color: #3E4072;
}
#label_Title{
 color: #8F8F8F;
 font-weight: 600;
}
QLineEdit {
 background-color:#DFDFDF;
 border-radius: 3px;
}
#pushButton_Confirm,
#pushButton_Cancel {
 border: none;
 border-radius: 8px;
}
#pushButton_Confirm {
 color: white;
 background-color: #3E4072;
}
#pushButton_Confirm:hover {
 color: white;
 background-color: #595D89;
}
#pushButton_Cancel {
 background-color: #D8D8D8;
}
#pushButton_Cancel:hover {
 background-color: #ACACAC;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Title">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="text">
      <string>Update progress</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="0" column="1">
       <widget class="QLabel" name="label_FileCount">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Downloading file...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_FileName">
        <property name="text">
         <string>--</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
